README.md

```diff
@@ -72,6 +72,11 @@ class APITestCase(TestCase):
 If you want to update the snapshots automatically you can use the `python manage.py test --snapshot-update`.
 Check the [Django example](https://github.com/syrusakbary/snapshottest/tree/master/examples/django_project).
 
+## Disabling terminal colors
+
+Set the environment variable `ANSI_COLORS_DISABLED`.
```

Comment:
I think it would be helpful to make it a bit clearer how and where, e.g.:

> To disable terminal colors, run `ANSI_COLORS_DISABLED=1 pytest`.

Thoughts?